tests/macos/xcodeproj/tests.sh

```diff
@@ -2,6 +2,9 @@ set -eux
 
 cd $(dirname $0)
 
+# Make sure there are simulators avilable as destinations
+xcrun simctl list
```

Comment:
What will happen if there are no available sims? It seems like this command just lists the available sims. It doesn't error out the script or create any new sims, in the case that there are no sims available.

Reply:
this implicitly creates the simulators if not exist, so that the next step is smooth and won't end up having no simulators to use, i have updated the comment here